RandBLAS/trig_skops.hh

```diff
@@ -0,0 +1,538 @@
+#include "RandBLAS/base.hh"
+#include "RandBLAS/exceptions.hh"
+#include "RandBLAS/random_gen.hh"
+#include "RandBLAS/sparse_skops.hh"
+#include "util.hh"
+
+#include <Random123/philox.h>
+#include <blas.hh>
+
+#include <iostream>
+#include <stdio.h>
+#include <stdexcept>
+#include <string>
+#include <sys/_types/_int64_t.h>
+#include <tuple>
+
+#include <math.h>
+#include <typeinfo>
+#include <vector>
+
+#define MAX(a, b) (((a) < (b)) ? (b) : (a))
+#define MIN(a, b) (((a) < (b)) ? (a) : (b))
+
+namespace RandBLAS {
+    // =============================================================================
+    /// WARNING: None of the following functions or overloads thereof are part of the
+    /// public API
+    ///
```

Comment:
Don't put things in the top-level namespace if they aren't part of the public API. Use a nested namespace. Example: https://github.com/BallisticLA/RandBLAS/blob/fd180c5743757936fdfa5be8b0fecc62d66b55c4/RandBLAS/sparse_data/csc_trsm_impl.hh#L40